server/server/routes.py

```diff
@@ -1,5 +1,7 @@
 def includeme(config):
     config.add_static_view('static', 'static', cache_max_age=3600)
     config.add_route('home', '/')
+    config.add_route('recover_password', '/recover_password')
```

Comment:
Don't use underline in url. Better to use '-'

Reply:
Underscores in URL's path are ok. But they are not allowed in domain names.